### DISCUSSION TITLE
游戏崩溃

### DISCUSSION THREAD

**Discussion by 147183** [Meloong-Git/PCL#1897] [提问] [answered]
[错误报告-2023-5-26_18.52.27.zip](https://github.com/Hex-Dragon/PCL2/files/11574444/-2023-5-26_18.52.27.zip)
我打开版本隔离后，还加了个灾变模组，游戏就崩溃了

**Comment by MoYuan-CN** [comment]


**Reply by 147183** [reply]
怎么弄呀

**Comment by MoYuan-CN** [comment, marked as answer]
~~搞反了（悲~~
是 `cataclysm` 需要 `citadel` 1.8.0或以上
更新下模组就可以啦

**Reply by 147183** [reply]
麻烦你了，谢谢

**Reply by MoYuan-CN** [reply]
没事
~~我看 #1896 还活着，关了吧~~
